frontend/nexus/post.ts

```diff
@@ -144,7 +145,7 @@ schema.extendType({
           skip: args.skip,
           first: args.first,
           orderBy: {
-            createdAt: 'desc',
+            publishedAt: 'desc',
```

Comment:
how does this handle `null` in the publishedAt col? Do they end up at the start or end of the list?

Reply:
They end up at the beginning, that's why I'm proposing that we populate the `publishedAt` column with the same value as `createdAt` for all existing posts that have a `status` of `'PUBLISHED'` so that we can preserve the same ordering as there is today. Does that sound good to you?

Reply:
yup, that works for me 👍